README.md

```diff
@@ -1,97 +1,45 @@
+![Mobile preview](./files/images/mobile.png)
+### <img src="./files/images/schedule_content_g_0.png" alt="Storytelling" width="2%"/> Schedule content
+Plan at which day and time articles should be published – and when they shouldn’t be visible to the public anymore.
+### <img src="./files/images/flexible_system_b_0.png" alt="Storytelling" width="2%"/> Flexible system
+Benefit from a wide range of additional functionality by industry partners or by one of more than 2,500 modules from the Drupal community.
+### <img src="./files/images/drupal_community_b_1.png" alt="Storytelling" width="2%"/> Open Source
+Using open source technology reduces development costs and resources and makes you part of the Drupal community with more than 100,000 users actively contributing.
 
 ## Installation
-
-Download thunder from the [distribution page](https://www.drupal.org/project/thunder) or just do:
-
-```
-drush dl thunder
-```
-
-If you want to install Thunder with composer (recommended) do the following:
+**Highly recommended: Install Thunder with composer**
 
 ```
 composer create-project burdamagazinorg/thunder-project thunder --no-interaction
 ```
+For further information follow the [install instructions](https://www.drupal.org/docs/8/install).
 
-For further information follow the [install instructions](https://www.drupal.org/docs/8/install). 
-
-## Configuration
+If you don't want to use Composer, you can install Thunder the traditional way by downloading a tarball from our drupal.org project page.
+## Issues & Contribution
+If you found an issue with Thunder, please open a [ticket](https://www.drupal.org/project/issues/thunder?categories=All) on drupal.org. Please have in mind that Thunder is a collection of many drupal contrib modules. So if you could track down an issue to a specific module, please open the ticket in the corresponding issue queue.
 
-In Thunder's default configuration, the automated cron module is disabled. For correct functioning of non-local sites, crons should be configured properly.
-There are two types of crons which should be configured:
+If you want to help, to make Thunder even better, open an issue on [drupal.org](https://www.drupal.org/project/issues/thunder?categories=All) or just create a pull request.
 
-* [Configure cron](https://www.drupal.org/docs/7/setting-up-cron-for-drupal/configuring-cron-jobs-using-the-cron-command)
-* [Configure the schedulers cron](http://cgit.drupalcode.org/scheduler/tree/README.txt?h=8.x-1.x) (Used for automated article publishing)
+## Documentation
+For more information please take a look at [https://burdamagazinorg.github.io/thunder-documentation/](https://burdamagazinorg.github.io/thunder-documentation/).
 
 ## Development
-
-For Development information please take a look at [docs/development.md](docs/development.md).
+For Development information please take a look at [docs/development.md](docs/development.md).
```

Comment:
Besser: "For information on the installation and development of Thunder, please ..."